txdbus/authentication.py

```diff
@@ -43,7 +44,9 @@ def beginAuthentication(self, protocol):
         
 
     def handleAuthMessage(self, line):
-        if not b' ' in line:
+        if not isinstance(line, six.binary_type):
+            line = six.b(line)
```

Comment:
No good `six.b` is implemented as `line.encode("latin-1")`. Please either throw in this case or use `.encode("ascii")` since that is [what the spec mandates](https://dbus.freedesktop.org/doc/dbus-specification.html#auth-protocol):

> The [authentication] protocol is a line-based protocol, where each line ends with \r\n. Each line begins with an all-caps ASCII command name containing only the character range [A-Z_], a space, then any arguments for the command, then the \r\n ending the line. The protocol is case-sensitive. All bytes must be in the ASCII character set.